### PR TITLE
Kataphract Team Update

### PR DESCRIPTION
**Pull request by Geevies**
* Added a Klaxan Kataphract Hopeful to the Kataphract response team.
* The Kataphracts now start with better and more stylized gear.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: html/changelogs/geeves - kataphractRedux.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: ChangeMe
```
Comment (fernerr) [inline html/changelogs/geeves - kataphractRedux.yml]:
```suggestion
author: Geeves
```

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -209,6 +209,17 @@
 	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
 	siemens_coefficient = 0.35
 
+/obj/item/clothing/head/helmet/unathi/klax
+	name = "klaxan hopeful helmet"
+	desc = "A helmet designated to be worn by a K'lax hopeful, the retrofit is only a bit shoddy."
```
Comment (alsoandanswer) [inline code/modules/clothing/head/helmet.dm]:
```suggestion
	desc = "A helmet designated to be worn by a K'lax hopeful. The retrofit is only a bit shoddy."
```

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -520,6 +520,19 @@
 	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
 	siemens_coefficient = 0.35
 
+// Vaurca version of Unathi armour
+/obj/item/clothing/suit/armor/unathi/klax
+	name = "klaxan hopeful body armor"
+	desc = "An armored chestplate designated to be worn by a K'lax hopeful, the retrofit is only a bit shoddy."
```
Comment (alsoandanswer) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
	desc = "An armored chestplate designated to be worn by a K'lax hopeful. The retrofit is only a bit shoddy."
```

**Comment by runecap** [comment]
Will the K'laxan Hopeful be whitelisted or open to everyone?

**Comment by Geevies** [comment]
The K'lax will be whitelisted due to how difficult they are to roleplay compared to a standard honourable unathi.

**Pull request merged**